conf/defaults.ini

```diff
@@ -860,6 +866,18 @@ max_concurrent_screenshots = 5
 # screenshots will be persisted to disk for up to temp_data_lifetime.
 upload_external_image_storage = false
 
+#################################### Alerting ############################
```

Comment:
I don't understand where these changes are from, either. I must need to do a better resolution with main.